actions/structures.xml

```diff
@@ -45,6 +39,9 @@
                     <action id="item-class-delete" name="Delete" url="/taoItems/Items/deleteClass" context="class" group="tree" binding="removeNode" >
                         <icon id="icon-bin"/>
                     </action>
+                    <action id="item-delete-all" name="Delete" url="/taoItems/Items/deleteAll" context="resource" multiple="true" group="tree" binding="removeNodes">
```

Comment:
The display name of 'delete all' is the same as for the 'delete one'.

Reply:
They aren't supposed to be displayed at the same time, multiple and single contexts are exclusives. The bug is fixed in tao-core